server/views/pages/plan-summary.njk

```diff
@@ -177,6 +176,18 @@
                     {% set goalType = 'current' %}
                 {% endif %}
 
+                {% if goalType == 'current' and goals.length == 0 %}
+                    <p>{{ locale.ifNoCurrentGoals.text }}</p>
+                    <ul>
+                        <li><a href="/create-goal/accommodation">{{ locale.ifNoCurrentGoals.action1 }}</a></li>
+                        <li><a href="/about-pop">{{ locale.ifNoCurrentGoals.action2 }}</a></li>
+                    </ul>
+                {% endif %}
+
+                {% if goalType == 'future' and goals.length == 0 %}
```

Comment:
Can you turn this into an elseif so that there's only one `if` statement in this new code?